models/collection.js

```diff
@@ -53,6 +57,7 @@ module.exports = {
     get: Joi.object(mutate(GET_MODEL, [
         'id',
         'name',
+        'pipelineIds',
```

Comment:
The [collectionFactory](https://github.com/screwdriver-cd/models/blob/master/lib/collectionFactory.js#L40) sets it to an empty array by default